README.md

```diff
@@ -227,6 +235,48 @@ clean environment. It can be omitted.
 
 ---
 
+## 🤝 Contributing
+
+We welcome contributions from the community! Here's how you can help:
+
+1. Fork the repository
+2. Create your feature branch (`git checkout -b feature/amazing-feature`)
+3. Commit your changes (`git commit -m 'Add some amazing feature'`)
+4. Push to the branch (`git push origin feature/amazing-feature`)
+5. Open a Pull Request
+
+Please read our [Contributing Guidelines](CONTRIBUTING.md) for more details.
+
+## 🔍 Troubleshooting
+
+Common issues and their solutions:
+
+1. **Connection Issues**
+   - Verify your network connectivity
+   - Check if the required ports are open
+   - Ensure your Bitcoin node is fully synced
+
+2. **Build Errors**
+   - Make sure all prerequisites are installed
+   - Update to the latest Go version
+   - Clear your build cache and try again
+
+For more detailed troubleshooting, please check our [Wiki](https://github.com/hemilabs/heminetwork/wiki) or open an issue.
+
+## 📚 Additional Resources
+
+- [Official Documentation](https://docs.hemi.network)
+- [API Reference](https://api.hemi.network)
+- [Community Forum](https://forum.hemi.network)
+- [Discord Community](https://discord.gg/hemi)
+
 ## 📄 License
 
 This project is licensed under the [MIT License](https://github.com/hemilabs/heminetwork/blob/main/LICENSE).
+
+---
+
+<div align="center">
+  <p>Built with ❤️ by the Hemi Labs team and contributors.</p>
+  <p>© 2024 Hemi Labs. All Rights Reserved.</p>
```

Comment:
Wrong year.

Reply:
And the contents of this repository are MIT Licensed, not ARR.